assets/js/analytics-advanced-tracking.js

```diff
@@ -33,11 +32,10 @@ import setUpAdvancedTracking from './analytics-advanced-tracking/set-up-advanced
  * @param {Object} metadata Additional event metadata to send, or `null`.
  */
 function sendEvent( action, metadata ) {
-	if ( ! metadata ) {
-		global.gtag( 'event', action );
-	} else {
-		global.gtag( 'event', action, metadata );
-	}
+	window.gtag( 'event', action, metadata || undefined ); // eslint-disable-line no-restricted-globals
```

Comment:
Why not `global` like elsewhere?

Reply:
@felixarntz it prevents webpack module concatenation optimization if we use global variables.